scripts/apps/relations/directives/RelatedItemsDirective.ts

```diff
@@ -48,7 +48,13 @@ export function RelatedItemsDirective(authoringWorkspace, relationsService) {
                     const type = getSuperdeskType(event);
                     const item = angular.fromJson(event.originalEvent.dataTransfer.getData(type));
 
-                    scope.addRelatedItem(item);
+                    const isDuplicate = Object.keys(scope.item.associations || {})
```

Comment:
Rename to `itemAlreadyAddedAsRelated`. It would also be useful to prevent adding self as related.

Reply:
done